docs/sources/installation/debian.md

```diff
@@ -1,40 +1,45 @@
 +++
-title = "Installing on Debian / Ubuntu"
-description = "Install guide for Grafana"
+title = "Install on Debian/Ubuntu"
+description = "Install guide for Grafana on Debian or Ubuntu"
 keywords = ["grafana", "installation", "documentation"]
 type = "docs"
 aliases = ["/installation/installation/debian"]
 [menu.docs]
-name = "Installing on Ubuntu / Debian"
+name = "Install on Ubuntu/Debian"
 identifier = "debian"
 parent = "installation"
 weight = 200
 +++
 
-# Installing on Debian / Ubuntu
+# Install on Debian or Ubuntu
 
-Read [Upgrading Grafana]({{< relref "installation/upgrading.md" >}}) for tips and guidance on updating an existing
-installation.
+This page explains what the installation package does, how to install Grafana dependencies, download and install Grafana, and then get the service up and running on your system.
 
-## Download
+**Note on upgrading:** While the process for upgrading Grafana is very similar to installing Grafana, there are some key backup steps you should perform. Read [Upgrading Grafana]({{< relref "installation/upgrading.md" >}}) for tips and guidance on updating an existing installation.
 
-Go to the [download page](https://grafana.com/grafana/download?platform=linux) for the latest download
-links.
+## Package details
 
-```bash
-wget <debian package url>
-sudo apt-get install -y adduser libfontconfig1
-sudo dpkg -i grafana_<version>_amd64.deb
-```
+- Installs binary to `/usr/sbin/grafana-server`
+- Installs Init.d script to `/etc/init.d/grafana-server`
+- Creates default file (environment vars) to `/etc/default/grafana-server`
+- Installs configuration file to `/etc/grafana/grafana.ini`
+- Installs systemd service (if systemd is available) name `grafana-server.service`
+- The default configuration sets the log file at `/var/log/grafana/grafana.log`
+- The default configuration specifies an sqlite3 db at `/var/lib/grafana/grafana.db`
+- Installs HTML/JS/CSS and other Grafana files at `/usr/share/grafana`
+
+## 1. Download and install
 
-You will find package URLs on the [download page](https://grafana.com/grafana/download?platform=linux).
+We recommend  you run all the listed commands before you download and install Grafana. They might not be necessary on all systems, but if you run them first then you will not be interrupted by dependency errors.
 
-## APT Repository
+You can install Grafana using an APT repository, by downloading a .deb package, or by using a binary .tar file.
 
-The command `add-apt-repository` isn't a default app on Debian 9 and requires you to run:
+### Install from APT repository 
+
+The command `add-apt-repository` is not a default app on Debian 9 and requires you to run:
```

Comment:
Unclear on Line 39. Are we saying that Debian 9 users must install from the APT repository? Or just that if they want to install from the APT repository, that they must run this command?

Reply:
no, we are saying to add the repository they need to run that. But I think we can remove the `sudo apt-get install -y software-properties-common` line , and move the
`You may need to install the apt-transport-https package on Debian before proceeding:
sudo apt-get install apt-transport-https`

line up to take it's place